com.unity.render-pipelines.high-definition/Runtime/Material/Unlit/UnlitData.hlsl

```diff
@@ -30,7 +30,7 @@ void GetSurfaceAndBuiltinData(FragInputs input, float3 V, inout PositionInputs p
 
     // Builtin Data
     ZERO_BUILTIN_INITIALIZE(builtinData); // No call to InitBuiltinData as we don't have any lighting
-    builtinData.opacity = alpha;
+    builtinData.opacity = lerp(_AlphaRemapMin, _AlphaRemapMax, alpha);
```

Comment:
This is not correct, alpha need to be modify at the source here, in 
float alpha = SAMPLE_TEXTURE2D(_UnlitColorMap, sampler_UnlitColorMap, unlitColorMapUv).a * _UnlitColor.a;
=>
float alpha = SAMPLE_TEXTURE2D(_UnlitColorMap, sampler_UnlitColorMap, unlitColorMapUv).a * _UnlitColor.a;
alpha = lerp(_AlphaRemapMin, _AlphaRemapMax, alpha);

as it is used in the alpha test after that GENERIC_ALPHA_TEST(alpha, _AlphaCutoff); . Same is true for lit shader

Reply:
Open question:
That should be:
float alpha = SAMPLE_TEXTURE2D(_UnlitColorMap, sampler_UnlitColorMap, unlitColorMapUv).a * _UnlitColor.a;
alpha = lerp(_AlphaRemapMin, _AlphaRemapMax, alpha);
Or
float alpha = SAMPLE_TEXTURE2D(_UnlitColorMap, sampler_UnlitColorMap, unlitColorMapUv).a
alpha = lerp(_AlphaRemapMin, _AlphaRemapMax, alpha);
alpha *= _UnlitColor.a;

The remap should be on data only?